tests/compiler/closure.ts

```diff
@@ -13,12 +13,52 @@ function testVar(): (value: i32) => i32 {
 }
 testVar();
 
-function testLet(): (value: i32) => i32 {
-  let $local0 = 0;
-  return function inner(value: i32) {
-    return $local0; // closure
-  };
+//function testLet(): (value: i32) => i32 {
+  //let $local0 = 0;
+  //return function inner(value: i32) {
+    //return $local0; // closure
+  //};
+//}
+//testLet();
```

Comment:
This still doesn't work, not sure if it's something that should work or not. Once I know I'll either fix this or move this into a separate test to test against stderr

Reply:
Moving it into limitations for now, since it doesn't seem necessary for an mvp